core-models/src/main/java/com/anytypeio/anytype/core_models/ext/StructExt.kt

```diff
@@ -37,4 +39,10 @@ fun Map<Id, Struct>.unset(
     val curr = getOrDefault(target, emptyMap())
     val update = curr - keys
     return this + mapOf(target to update)
+}
+
+fun Struct?.mapToObjectWrapperType(): ObjectWrapper.Type? {
+    if (this == null) return null
+    if (!this.containsKey(Relations.ID) || !this.containsKey(Relations.UNIQUE_KEY)) return null
```

Comment:
`if (contains(Relations.ID) && contains(Relations.UNIQUE_KEY) return ObjectWrapper.Type(this), else return null ` would be more readable

Reply:
done